File: LP SWI-Prolog/Clase 20 - Semana 10 - 02 Octubre/Ejercicio 2/Forma 1.pl
% penultimo(+L, ?P)
penultimo([P,_], P). % caso base
penultimo([_|R], X) :- penultimo(R, X). % caso recursivo

/**
 * Autor(es): 
 * - Joy Bonilla
 * - Natalia Solano
 * - Javier Amador D.
 * - Kevin Flores G.
 */

File: Programación Funcional/Clase 11 - Semana 06 - 01 Setiembre/Ejercicio 1/Forma 1.java

BinaryOperator<Integer> swapArgs( BinaryOperator<Integer> f ) {
return (x,y) -> f.apply(y,x); 
}

/* Autor(es):
     Javier Alberto Amador Delgado
     Kevin Flores García
     Natalia Solano Azofeifa
     David Alberto Guevara Sánchez
     Joy Bonilla Fley
*/
